packages/notifi-react/README.md

```diff
@@ -1,81 +1,139 @@
 # Notifi React SDK
 
-`@notifi-network/notifi-react` accelerates the development of Notifi integrated dapps by providing:
+`@notifi-network/notifi-react` accelerates the development of Notifi integrated dapps.
 
-- Out-of-the-box React components (`NotifiCardModal`) by which developers can just plug and play.
+-  [Out of The Box Implementation](#out-of-the-box-implementation)
+-  [Customized Implementation](#advanced-customized-implementation)
+-  [Creating Your Own Card](#creating-your-own-card)
+-  [Migrating from an older version of the SDK ](#migrate-your-card)
 
-- React Contexts for more advanced use cases in case developers want to build their own UI components.
 
-> Give it a hand-on by checking out the [example project](https://notifi-sdk-ts-vercel-notifi-react-example-v2.vercel.app/)
 
-## React Components (NotifiCardModal)
+> Play around with a Card [Example](https://notifi-sdk-ts-vercel-notifi-react-example-v2.vercel.app/)
 
-Dapp developers can easily mount the `NotifiCardModal` component to their dapp to enable the Notifi notification service without any hassle. The `NotifiCardModal` component provides the following features:
+<br/><br/>
 
-1. login Notifi services
+# Out of The Box Implementation
 
-2. manage (subscribe/unsubscribe) topics
+Dapp developers can easily mount the `NotifiCardModal` component to their dapp to enable the Notifi notification service. The `NotifiCardModal` component provides the following features to the user:
 
-3. manage (add/remove) targets
+- Login to Notifi Services
 
-4. Preview notification details
+-  Manage Subscriptions to Notifications
 
-### Getting Started
+- Manage Targets/Channels for Notifications
 
-1. Make sure the react environment is set up properly with the following dependencies:
+- Preview Notification Details
 
+<br/><br/>
+## Getting Started
+*Environment*
 - Node version >= 18 (with corresponding npm version)
 - React version >= 17
 
-2. Make sure you have a Notifi tenant account. we will need the following information to set up the `NotifiCardModal`.
+>If you haven't created a Notifi tenant account yet [set up an account](https://admin.notifi.network/signup?environment=prd)
 
-- tenantId
-- cardId
-
-> You can register a new tenant through [Notifi Admin Portal](https://admin.notifi.network/) for free if you do not have one yet.
-> Check out the [detail here](https://docs.notifi.network/docs/getting-started)
-
-#### Mount `NotifiCardModal` to your dapp
-
-1. Wrap your app with `NotifiContextProvider`
+<br/><br/>
+### Mount the `NotifiCardModal` to your dApp
 
+*Example*
 ```tsx
-import { NotifiFrontendClientProvider } from '@notifi-network/notifi-react';
+import { NotifiContextProvider, NotifiCardModal } from '@notifi-network/notifi-react';
+import { useEthers } from '@usedapp/core';
+import { providers } from 'ethers';
+
+const NotifiCard = () => {
```

Comment:
Do you think we also need to mention that we need to wrap the card with `NotifiContextProvider` so that we are not misleading users

Reply:
I'll mention it here at the highest level